### ISSUE TITLE
Upates to javaparser-maven-sample/README.md

### ISSUE THREAD

**Issue by nchamberlain** [javaparser/javaparser-maven-sample#83]
As a complete nooB at Maven and JavaParser, this sample project sounded perfect for me. However, I stumbled a lot trying to use it. I would like to make a few suggestions _(highlighted in italics)_ to this README:

On line 4, please make two changes so the sentence reads:
_This_ fully working sample Maven project parses and generates code with [JavaParser](http://www.javaparser.org)
**Reason:** the existing line 4 is a sentence fragment. As a 30 yr tech writer/editor, I feel the overwhelming need to read/write complete sentences. Changing the first word and deleting the seventh word ("that") will make this a complete sentence with minimal impact to the original wording.

On line 6, please add the word "sample" between the first and second words so it reads:
This _sample_ is targeted at people without [Maven](https://maven.apache.org/) experience..
**Reason:** adding the word "sample" makes the word "this" a little more specific. Grammatically, the word "this" refers to the last noun written which is the word "code".

Between lines 10 and 11, please add instructions similar to:
_If required, install git <insert link to git here>. If you haven't already done so, clone this sample repository with `git clone https://github.com/javaparser/javaparser-maven-sample.git`. Change to the root folder of where this sample project was installed (where the pom.xml file is located)._
**Reason:** As a Maven nooB, I didn't know if I was supposed to manually clone this sample project or if Maven would somehow auto-magically figure out what repository dependency I had. I finally decided that because I needed the pom.xml file that I needed to git a clone of this sample myself and run mvn from inside the main/top/root folder of this sample. I know, I know, **now** it seems so obvious but I am a double nooB, both git and Maven.

Add to line 14:
_This runs LogicPositivizer which reads, parses, and modifies the code in resources/Blabla.java and then writes the modified file to the output folder._
**Reason:** as a complete nooB, I had wildly wrong expectations about what this sample would do or produce. It was only after drilling down in the src tree and reading thru LogicPositivizer.java and Blabla.java that I appreciated what had happened. It sort of reminded me of a chemistry experiment where a material is added to the clear liquid in a test tube and the liquid remains clear and apparently unaffected afterwards, only to find out later that a clear odorless gas was output. I didn't understand and appreciate what had just happened. **Maybe** even at end of LogicPositivizer.java, insert a `System.out.println("parsed Blabla.java and wrote modified file to output folder.");` statement to tell user that something happened.

Somewhere after line 16, please add a line similar to:
_Please refer to "JavaParser: Visited" <insert link to book here> for more information about what the code in LogicPositivizer.java does._
**Reason:** If someone jumps into this sample completely unaware of how powerful JavaParser is, they may be confused or mystified by the sample code.

**Comment by jlerbsc** [comment]
Can you suggest an RP for your issue?

**Comment by nchamberlain** [comment]
I'm sorry, I'm a nooB at git and maven: is "RP" short for "repository" or "pull request" or something else? Thanks for you patience.

**Comment by jlerbsc** [comment]
Sorry, this is a pull request. To modify the documentation you don't need to use maven. But if you don't feel up to it I can make the changes directly for you.

**Comment by nchamberlain** [comment]
Well, I failed to create a pull request, I think because I am not a member of this project. Sorry I couldn't do it.
Please make the changes for me. I tried to attach an updated copy of the README.md so you don't have to repeat making the changes. 
[README.md](https://github.com/javaparser/javaparser-maven-sample/files/14108840/README.md)

**Comment by nchamberlain** [comment]
So I forked the repository, updated the README.md file, committed it and then did a pull request and I believe it worked! Thanks for all your help and patience and let me know if I didn't do it correctly or if there was something else that needs to be done

**Comment by jlerbsc** [comment]
Your contribution has been accepted.

**Comment by nchamberlain** [comment]
Thanks again for your help
Nelson
